src/plugins/intel_cpu/tests/functional/single_layer_tests/classes/eltwise.cpp

```diff
@@ -63,7 +63,26 @@ ov::Tensor EltwiseLayerCPUTest::generate_eltwise_input(const ov::element::Type&
             break;
         }
     } else {
-        params = gen_params(INT32_MAX, INT32_MIN);
+        switch (type) {
```

Comment:
For me it makes sense to extract Bitwise operations coverage into separate test class like it is done for [Logical ops](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/functional/single_layer_tests/logical.cpp). Reuse part with Eltwise test is minimal, just increases complexity.
It also allows us to avoid InputLayerType::None type introduction.

Reply:
I partially agree with you. As variant we can refactor `EltwiseLayerCPUTest` and extract shared part in shared test class (src\tests\functional\shared_test_classes). But right now, I prefer keep it as is. Some thoughts:
1) Bitwise operation is element wise operation.
2) The CPU plugin implements several dozen eltwise operations. Each of them is verified by functional test `EltwiseLayerCPUTest`. Why should an engineer write a test for a new eltwise operation from scratch? As I understood there is a test for that: `EltwiseLayerCPUTest`.
3) If I implement test for new element-wise bitwise operations from scratch then I should repeat the same functionality in bitwise test:
   * `selectedType` (selected primitive) update. `LogicalLayerCPUTest` does it.
   * `opType` (SCALAR/VECTOR) handling. `LogicalLayerCPUTest` does not handle it, but we need do it for bitwise.
   * `secondaryInputType` (PARAMETER/CONSTANT) handling (`LogicalLayerCPUTest` does it )
   * `generate_eltwise_input` will be reused

Let's discuss details offline.

Reply:
thanks. discussed offline, decided to merge as is